diff --git a/node_modules/react-native-google-places/android/build.gradle b/node_modules/react-native-google-places/android/build.gradle
index 8e2c769..992c1df 100644
--- a/node_modules/react-native-google-places/android/build.gradle
+++ b/node_modules/react-native-google-places/android/build.gradle
@@ -46,7 +46,13 @@ dependencies {
 
     compileOnly "com.facebook.react:react-native:+"
     implementation "com.google.android.libraries.places:places:1.1.0"
-    implementation "com.android.support:appcompat-v7:${supportLibVersion}"
-    implementation "com.android.support:support-v4:${supportLibVersion}"
     implementation "com.google.code.findbugs:jsr305:3.0.2"
+
+    if (rootProject.findProperty("android.useAndroidX")) {
+        implementation "androidx.appcompat:appcompat:1.0.2"
+        implementation "androidx.legacy:legacy-support-v4:1.0.0"
+    } else {
+        implementation "com.android.support:appcompat-v7:${supportLibVersion}"
+        implementation "com.android.support:support-v4:${supportLibVersion}"
+    }
 }
\ No newline at end of file
