diff --git a/node_modules/react-native-storage-wrapper/src/index.js b/node_modules/react-native-storage-wrapper/src/index.js
index d5e6d37..a58153f 100644
--- a/node_modules/react-native-storage-wrapper/src/index.js
+++ b/node_modules/react-native-storage-wrapper/src/index.js
@@ -1,4 +1,4 @@
-import {AsyncStorage} from "react-native";
+import AsyncStorage from '@react-native-community/async-storage';
 
 module.exports = {
     get (key) {
