diff --git a/node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java b/node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java
index ab82f50..5b7a69c 100644
--- a/node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java
+++ b/node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java
@@ -376,10 +376,7 @@ public class RNDeviceModule extends ReactContextBaseJavaModule {
   public void isLocationEnabled(Promise p) {
       boolean locationEnabled = false;
 
-      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-        LocationManager mLocationManager = (LocationManager) reactContext.getApplicationContext().getSystemService(Context.LOCATION_SERVICE);
-        locationEnabled = mLocationManager.isLocationEnabled();
-      } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
         int locationMode = Settings.Secure.getInt(reactContext.getContentResolver(), Settings.Secure.LOCATION_MODE, Settings.Secure.LOCATION_MODE_OFF);
         locationEnabled = locationMode != Settings.Secure.LOCATION_MODE_OFF;
       } else {
