diff --git a/node_modules/react-native-google-location/android/app/src/main/java/com/timhagn/rngloc/LocationProvider.java b/node_modules/react-native-google-location/android/app/src/main/java/com/timhagn/rngloc/LocationProvider.java
index cc23b61..4770a05 100644
--- a/node_modules/react-native-google-location/android/app/src/main/java/com/timhagn/rngloc/LocationProvider.java
+++ b/node_modules/react-native-google-location/android/app/src/main/java/com/timhagn/rngloc/LocationProvider.java
@@ -103,6 +103,7 @@ public class LocationProvider implements
             LocationServices.FusedLocationApi.removeLocationUpdates(mGoogleApiClient, this);
             mGoogleApiClient.disconnect();
         }
+        connected = false;
     }
 
     @Override
@@ -116,7 +117,12 @@ public class LocationProvider implements
             mLocationCallback.handleNewLocation(location);
         }
         // Now request continuous Location Updates
-        LocationServices.FusedLocationApi.requestLocationUpdates(mGoogleApiClient, mLocationRequest, this);
+        try {
+            LocationServices.FusedLocationApi.requestLocationUpdates(mGoogleApiClient, mLocationRequest, this);
+        } catch (Exception e) {
+            e.printStackTrace();
+            disconnect();
+        }
     }
 
     @Override
diff --git a/node_modules/react-native-google-location/android/app/src/main/java/com/timhagn/rngloc/RNGLocationModule.java b/node_modules/react-native-google-location/android/app/src/main/java/com/timhagn/rngloc/RNGLocationModule.java
index 137fef2..d21ec5f 100644
--- a/node_modules/react-native-google-location/android/app/src/main/java/com/timhagn/rngloc/RNGLocationModule.java
+++ b/node_modules/react-native-google-location/android/app/src/main/java/com/timhagn/rngloc/RNGLocationModule.java
@@ -1,7 +1,7 @@
 package com.timhagn.rngloc;
 
 import android.location.Location;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
 import android.util.Log;
 
 import com.facebook.react.bridge.Arguments;
@@ -42,14 +42,34 @@ public class RNGLocationModule extends ReactContextBaseJavaModule implements Loc
 
         // Check if all went well and the Google Play Service are available...
         if (!mLocationProvider.checkPlayServices()) {
+            mLocationProvider = null;
             Log.i(TAG, "Location Provider not available...");
         } else {
             // Connect to Play Services
-            mLocationProvider.connect();
+            //mLocationProvider.connect();
             Log.i(TAG, "Location Provider successfully created.");
         }
     }
 
+    @ReactMethod
+    public boolean available() {
+        if (mLocationProvider != null) return true;
+        return false;
+    }
+
+    @ReactMethod
+    public void disconnect() {
+        if (mLocationProvider != null && mLocationProvider.connected) {
+            mLocationProvider.disconnect();
+        }
+    }
+
+    @ReactMethod
+    public void reconnect() {
+        if (mLocationProvider != null && !mLocationProvider.connected) {
+            mLocationProvider.connect();
+        }
+    }
 
     @Override
     public String getName() {
diff --git a/node_modules/react-native-google-location/android/app/build.gradle b/node_modules/react-native-google-location/android/app/build.gradle
index 5b2e551..c83c065 100644
--- a/node_modules/react-native-google-location/android/app/build.gradle
+++ b/node_modules/react-native-google-location/android/app/build.gradle
@@ -30,7 +30,7 @@ repositories {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:0.12.+'
-  compile 'com.google.android.gms:play-services-location:8.3.0'
+  implementation 'com.facebook.react:react-native:0.12.+'
+  implementation 'com.google.android.gms:play-services-location:17.0.0'
 }
 
