diff --git a/node_modules/react-native-google-places/react-native-google-places.podspec b/node_modules/react-native-google-places/react-native-google-places.podspec
index 7445c14..870e1c2 100644
--- a/node_modules/react-native-google-places/react-native-google-places.podspec
+++ b/node_modules/react-native-google-places/react-native-google-places.podspec
@@ -19,6 +19,6 @@ Pod::Spec.new do |s|
   s.compiler_flags = '-DHAVE_GOOGLE_MAPS=1', '-fno-modules'
 
   s.dependency 'React'
-  s.dependency 'GooglePlaces', '~> 3.1.0'
-  s.dependency 'GoogleMaps', '~> 3.1.0'
+  s.dependency 'GooglePlaces', '~> 3.2.0'
+  s.dependency 'GoogleMaps', '~> 3.2.0'
 end
