diff --git a/node_modules/react-native-input-prompt/src/Prompt.js b/node_modules/react-native-input-prompt/src/Prompt.js
index 1f9353d..c9f57a9 100644
--- a/node_modules/react-native-input-prompt/src/Prompt.js
+++ b/node_modules/react-native-input-prompt/src/Prompt.js
@@ -14,6 +14,10 @@ import PropTypes from "prop-types";
 export default class Prompt extends Component {
 	static propTypes = {
 		title: PropTypes.string.isRequired,
+                aboveInputRender: PropTypes.func,
+                belowInputRender: PropTypes.func,
+		autoCorrect: PropTypes.bool,
+		autoCapitalize: PropTypes.string,
 		visible: PropTypes.bool.isRequired,
 		placeholder: PropTypes.string.isRequired,
 		cancelText: PropTypes.string,
@@ -30,6 +34,10 @@ export default class Prompt extends Component {
 
 	static defaultProps = {
 		title: "Title",
+                aboveInputRender: () => {},
+                belowInputRender: () => {},
+                autoCorrect: true,
+                autoCapitalize: "none",
 		visible: false,
 		placeholder: "Placeholder",
 		cancelText: "Cancel",
@@ -77,7 +77,6 @@
 		return (
 			<Modal
 				transparent={true}
-				animationType="fade"
 				visible={this.props.visible}
 				onRequestClose={this._onCancel.bind(this)}>
 				<View style={styles.screenOverlay}>
@@ -77,13 +85,18 @@ export default class Prompt extends Component {
 						<Text style={[styles.title, this.props.titleStyle]}>
 							{this.props.title}
 						</Text>
+                                                {this.props.aboveInputRender()}
 						<TextInput
+                                                        autoCorrect={this.props.autoCorrect}
+                                                        autoCapitalize={this.props.autoCapitalize}
 							placeholder={this.props.placeholder}
+                                                        defaultValue={this.props.value}
 							style={styles.textInput}
 							onChangeText={this._onChangeText.bind(this)}
 							onSubmitEditing={this._onSubmit.bind(this)}
 							autoFocus={true}
 						/>
+                                                {this.props.belowInputRender()}
 						<View style={styles.buttonsOuterView}>
 							<View style={styles.buttonsInnerView}>
 								<TouchableOpacity
diff --git a/node_modules/react-native-input-prompt/src/styles.js b/node_modules/react-native-input-prompt/src/styles.js
index b1be4c8..459bf1e 100644
--- a/node_modules/react-native-input-prompt/src/styles.js
+++ b/node_modules/react-native-input-prompt/src/styles.js
@@ -3,25 +3,22 @@ import { StyleSheet, Platform, Dimensions } from "react-native";
 export default StyleSheet.create({
 	screenOverlay: {
 		height: Dimensions.get("window").height,
-		backgroundColor: "black",
-		opacity: 0.8
+		backgroundColor: "rgba(0, 0, 0, 0.8)",
 	},
 	dialogPrompt: {
 		...Platform.select({
 			ios: {
-				opacity: 0.9,
-				backgroundColor: "rgb(222,222,222)",
-				borderRadius: 15
+				borderRadius: 15,
 			},
 			android: {
 				borderRadius: 5,
-				backgroundColor: "white"
 			}
 		}),
+                opacity: 1,
+		backgroundColor: "white",
 		marginHorizontal: 20,
 		marginTop: 150,
 		padding: 10,
-
 		flexDirection: "column",
 		justifyContent: "space-between",
 		alignItems: "center"
@@ -31,6 +28,15 @@ export default StyleSheet.create({
 		fontSize: 26,
 		color: "black"
 	},
+        aboveInputText: {
+                fontSize: 16,
+                color: "black"
+        },
+        belowInputText: {
+                fontSize: 16,
+                color: "black",
+		marginBottom: 15
+        },
 	textInput: {
 		height: 40,
 		width: "100%",
